packages/types/src/index.ts

```diff
@@ -1,3 +1,5 @@
+import './globals';
```

Comment:
This may be the fishiness you were talking about, but... how does this work?

Reply:
Take this with a grain of salt because I don't know a 100% either.

Fwiw we could also put `declare global { const __DEBUG_BUILD__: boolean; }` right here in the index.ts file. That would also work. However, (I think) we want it in a separate file so we need to somehow import it so that the ambient declaration `global` gets picked up when packages import from the types package, because the `"types"` entrypoint only points to `index.d.ts`.

`import './globals';` worked for consuming packages so I just used that. `import [type] {} from './globals'` did not work. Out of desperation I just tried `export type {} from './globals';`. That also seems to work and it's a little bit more in line with the other stuff we have in the package so I'm changing it to that (b2928d991f16002de5855d211691d18e5bcc0f0b). 

So yeah... that's how *I think* this whole thing works

Reply:
> `export type {} from './globals';`

So, in other words, this is a dummy export just to cause TS to load the `globals` file, which shoves `__DEBUG_BUILD__` into the ambient global namespace? (Which would explain how the import worked, also.)